Anlab.Mvc/Views/Admin/EditAdmin.cshtml

```diff
@@ -0,0 +1,66 @@
+@using AnlabMvc.Models.Roles
+@model AnlabMvc.Models.User.UserRolesModel
+
+@{
+    ViewData["Title"] = "Edit Admin Roles";
+}
+
+<div class="col">
+    <dl class="dl-horizontal">
+        <dt>
+            @Html.DisplayNameFor(model => model.User.Name)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.User.Name)
+        </dd>
+        <dt>
+            @Html.DisplayNameFor(model => model.User.Email)
+        </dt>
+        <dd>
+            @Html.DisplayFor(model => model.User.Email)
+        </dd>
+    </dl>
+
+    @if (Model.IsAdmin)
+    {
+        <form method="post" asp-action="AddUserToRole" asp-route-userId="@Model.User.Id" asp-route-role=@RoleCodes.Admin asp-route-add="false" asp-antiforgery="true" style="margin: 3px">
+            <button class="btn btn-red" type="submit"><i class="fa fa-trash"></i> Admin</button>
+        </form>
+    }
+    else
+    {
+        <form method="post" asp-action="AddUserToRole" asp-route-userId="@Model.User.Id" asp-route-role=@RoleCodes.Admin asp-route-add="true" asp-antiforgery="true" style="margin: 3px">
+            <button class="btn btn-default" type="submit"><i class="fa fa-plus"></i> Admin</button>
+        </form>
+    }
+    @if (Model.IsLabUser)

```

Comment:
@jSylvestre these are so close together, the only difference is one class and the route is false instead of true.  I'm thinking a quick partial would clean this up a ton